admin/tests/unit/components/organization-invitations_test.js

```diff
@@ -0,0 +1,31 @@
+import { module, test } from 'qunit';
```

Comment:
**remarque**
J'ai l'impression que les modifs ici n'auraient pas due atterrir dans le commit "[ADMIN] Move a member's invitation form to the invitations tab" ? 
En tout cas ce commit me paraît un peu fouilli comparés aux autres, c'est dommage c'était si beau à lire jusqu'à là 🤩